src/main.cpp

```diff
@@ -3041,13 +3042,13 @@ void UnlinkPrunedFiles(std::set<int>& setFilesToPrune)
 }
 
 /* Calculate the block/rev files that should be deleted to remain under target*/
-void FindFilesToPrune(std::set<int>& setFilesToPrune)
+void FindFilesToPrune(std::set<int>& setFilesToPrune, uint64_t nPruneAfterHeight)
```

Comment:
Nit: I think it'd make more sense if FindFilesToPrune() took a params object, as the PruneAfterHeight feels like an implementation specific detail that FlushStateToDisk() doesn't need to know about.